DuckDuckGo/Subscription/Views/SubscriptionFlowView.swift

```diff
@@ -64,8 +65,11 @@ struct SubscriptionFlowView: View {
         .applyInsetGroupedListStyle()
         .tint(Color(designSystemColor: .textPrimary))
         .environment(\.rootPresentationMode, self.$isActive)
+        .onAppear(perform: {
+            Pixel.fire(pixel: .privacyProOfferScreenImpression)
```

Comment:
Firing this should probably be responsibility of the view model.

Reply:
done